.gocd/build.gocd.groovy

```diff
@@ -135,7 +135,9 @@ GoCD.script {
           template = 'installers-gradle'
 
           materials {
-            add(ctx.repo)
+            add((ctx.repo as GitMaterial).dup({
+              shallowClone = false
```

Comment:
FYI you can also modify the `ctx.repo` directly without `dup()` if there’s no reason to work on a copy of the material config. The side effect is that will affect all pipelines that share `ctx.repo` config of course, rather than just that one pipeline, depending on what you’re going for.

```
ctx.repo.shallowClone = false
```

Reply:
Well, I didn't want to disable shallow clone for `build-linux-*`, `plugins-*` etc because those don't require older commits to do git log for.